src/gameengine/entities/Entity.java

```diff
@@ -21,7 +21,7 @@
     private int entityType = EntityType.STANDARD.ordinal();
     private int entityTypeBitMask = 1 << entityType;
     private Motion motion;
-    private Shape shape;
+    protected Shape shape;

```

Comment:
I was too lazy to think when creating the Wall class.  Fixed back to private thanks.